storagemanager/storage/views.py

```diff
@@ -26,9 +26,7 @@ class ShowAllListView(ListView):
     ordering = ['category', 'type', 'model']
 
 def show_items_from_category(request):
-    categories_set = set()                      
-    for item in Item.objects.all():
-        categories_set.add(item.category)
+    categories_set = sorted(Item.objects.values_list('category', flat=True).distinct())                     
```

Comment:
Nice :+1: 

Although, you don't need to fetch it through ORM - you could simply extract it from model with `Item.CATEGORIES` and some small in-line operation to fetch only values from tuples.